cmd/buildah/mkcw.go

```diff
@@ -0,0 +1,104 @@
+	mkcwDescription := `Convert a conventional image to a confidential workload image.`
+	mkcwCommand := &cobra.Command{
+		Use:   "mkcw",
+		Short: "Convert a conventional image to a confidential workload image",
+		Long:  mkcwDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			options.TeeType = define.TeeType(teeType)
+			return mkcwCmd(cmd, args, options)
+		},
+		Example: `buildah mkcw localhost/repository:typical localhost/repository:cw`,
+		Args:    cobra.ExactArgs(2),
+	}
+	mkcwCommand.SetUsageTemplate(UsageTemplate())
+	rootCmd.AddCommand(mkcwCommand)
+	flags := mkcwCommand.Flags()
+	flags.SetInterspersed(false)
+
+	flags.StringVarP(&teeType, "type", "t", "", "TEE type")
+	flags.StringVarP(&options.AttestationURL, "attestation-url", "u", "", "attestation server URL")
+	flags.StringVarP(&options.AttestationURL, "attestation_url", "", "", "attestation server URL (alternate flag spelling)")
+	if err := flags.MarkHidden("attestation_url"); err != nil {
+		panic("error marking attestation_url as hidden")
+	}
+	flags.StringVarP(&options.BaseImage, "base-image", "b", "", "alternate base image (default: scratch)")
+	flags.StringVarP(&options.BaseImage, "base_image", "", "", "alternate base image (default: scratch) (alternate flag spelling)")
+	if err := flags.MarkHidden("base_image"); err != nil {
+		panic("error marking base_image as hidden")
+	}
+	flags.StringVarP(&options.DiskEncryptionPassphrase, "encryption-passphrase", "p", "", "disk encryption passphrase")
+	flags.StringVarP(&options.DiskEncryptionPassphrase, "encryption_passphrase", "", "", "disk encryption passphrase (alternate flag spelling)")
+	if err := flags.MarkHidden("encryption_passphrase"); err != nil {
+		panic("error marking encryption_passphrase as hidden")
+	}
+	flags.IntVarP(&options.CPUs, "cpus", "c", 0, "number of CPUs to expect")
+	flags.IntVarP(&options.Memory, "memory", "m", 0, "amount of memory to expect (MB)")
+	flags.StringVarP(&options.WorkloadID, "workload-id", "w", "", "workload ID")
+	flags.StringVarP(&options.WorkloadID, "workload_id", "", "", "workload ID (alternate flag spelling)")
+	if err := flags.MarkHidden("workload_id"); err != nil {
+		panic("error marking workload_id as hidden")
+	}
+	flags.StringVarP(&options.Slop, "slop", "s", "25%", "extra space needed for converting a container rootfs to a disk image")
+	flags.StringVarP(&options.FirmwareLibrary, "firmware-library", "f", "", "location of libkrunfw-sev.so")
+	flags.StringVarP(&options.FirmwareLibrary, "firmware_library", "", "", "location of libkrunfw-sev.so (alternate flag spelling)")
+	if err := flags.MarkHidden("firmware_library"); err != nil {
+		panic("error marking firmware_library as hidden")
+	}
+	flags.BoolVarP(&options.IgnoreAttestationErrors, "ignore-attestation-errors", "", false, "ignore attestation errors")
+	flags.BoolVarP(&options.IgnoreAttestationErrors, "ignore_attestation_errors", "", false, "ignore attestation errors (alternate flag spelling)")
```

Comment:
Can we just do something shorter like --ignore?

Reply:
Feels like it would invite the question "ignore what?"  Keep in mind that if we don't manage to register the workload with an attestation server, the init process running in the VM can't ask the attestation server for the disk encryption passphrase.  So while an image that wasn't registered is still good enough to let our tests check that we formatted things correctly in a number of ways, the resulting image isn't usable beyond that.